### PR TITLE
Add convenience method to rollbar service for configuring person tracking

### LINKED ISSUE #29: Add support for setting person
Thanks for working on this, it looks really promising! We just got our app deployed with fastboot and realized rollbar.js breaks

Anyway, Rollbar supports [people tracking](https://docs.rollbar.com/docs/person-tracking), which has come in handy for me in the past. Not really sure how this would work in this addon, but I guess it can just be configured on the rollbar instance?

### PR DESCRIPTION
**Pull request by sdhull**
Pretty simple! Let me know if you'd like any changes. Thanks again for getting rollbar working with fastboot :100:

Fixes #29

### CODE REVIEW

**Comment by paulcwatts** [comment]
Thank you, I really appreciate it!

**Comment by bf4** [comment]
@paulcwatts This would make a nice point release :)

**Comment by paulcwatts** [comment]
@bf4, you're right, thanks for the reminder! I should be able to get to it this weekend.

**Comment by paulcwatts** [comment]
Released as 0.8.0.

File: addon/services/rollbar.js [hunk 1]
```diff
@@ -51,5 +51,8 @@ export default Service.extend({
   },
   critical() {
     return this.instance.critical(...arguments);
+  },
+  setPerson(person) {
+    return this.instance.configure({payload: {person}});
```
Comment (bf4) [inline addon/services/rollbar.js:56]: where is `configure` defined? Should this have been put in the `init` function as well?  When I inject the service and call `setPerson` I get `TypeError: this.instance.configure is not a function` pointing to this line

**Comment by bf4** [comment]
This seems to overwrite my payload: environment config.  Anyone using this have that issue?

Not sure if I should raise an issue yet.

```js
  ENV['rollbar'] = {
    enabled: environment === 'production',
    captureUncaught: true,
    payload: {
      environment: environment
    },
```

Since using setPerson, environment now comes through as unknown.

File: addon/services/rollbar.js (hunk 1 above)
Comment (sdhull) [reply addon/services/rollbar.js:56]: `configure` is defined on [`Rollbar` itself](https://docs.rollbar.com/docs/javascript#section-examples). Your issue is a good point actually, I failed to add the same method to the fake 'instance' that is supplied when rollbar is disabled, so I'll open a followup PR

**Comment by sdhull** [comment]
@bf4 I'll look into it today

**Pull request merged**